cassdegrees/templates/managecourses.html

```diff
@@ -0,0 +1,115 @@
+{% extends "base.html" %}
+
+{% block page-title %}Manage Courses{% endblock %}
+
+{% block subtitle %}Add, Modify or Delete Courses{% endblock %}
+
+{% block content %}
+    <form class="anuform" action="/manage_courses/" method="post">
+        {% csrf_token %}
+        <fieldset>
+            <legend>Add Course</legend>
+
+            <!-- As HTML forms only support get or post methods,
+            add this hidden input for consistency with the rest of the forms -->
+            <input type="hidden" name="_method" value="post">
+
+            <p>
+                <label for="id">Course Name:</label>
+                <input class="text" id="text" type="text" name="name" aria-required="true" required>
+            </p>
+
+            <p>
+                <label for="id">Course Code:</label>
+                <input class="text" id="text" type="text" name="code" minlength="8" maxlength="8" size="8" aria-required="true" required>
```

Comment:
```suggestion
                <input class="text tfull" id="text" type="text" name="code" minlength="8" maxlength="8" size="8" aria-required="true" required>
```

Same as above (for consistency)

Reply:
not particularly fond of having all the textboxes stretch to the other side of the page, especially when some input don't need to be that long. Having said that, after visiting [http://webpublishing.anu.edu.au/web-style-guide/forms.php](url) it seems like ANU likes it so I'm ok with this.

Reply:
Alternatively maybe you could reposition year and units to be at the bottom and leave those smaller, while making the other text fields/select fields larger. Might need further discussion with team